examples/tpm/tpm_key_helpers.h

```diff
@@ -0,0 +1,71 @@
+/* tpm_test_keys.h
+ *
+ * Copyright (C) 2006-2021 wolfSSL Inc.
+ *
+ * This file is part of wolfTPM.
+ *
+ * wolfTPM is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfTPM is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA
+ */
+
+#ifndef _TPM_TEST_KEYS_H_
+#define _TPM_TEST_KEYS_H_
+
+#ifndef WOLFTPM2_NO_WRAPPER
```

Comment:
All of the TPM support here requires `!WOLFTPM2_NO_WRAPPER`, so please make the build setting detection consistent for all TPM code. You can probably add a single well placed check:

```
#if defined(WOLFSSH_TPM) && defined(WOLFTPM2_NO_WRAPPER)
    /* wolfTPM support requires the wrapper to be enabled */
    #undef WOLFSSH_TPM
#endif
```

Reply:
Do you think this is required for the code in src/internal.c and ssh.c?

It is no longer required for examples/tpm/... of course, because the file is removed, and only the used code is in the client demo. I was wondering about src/internal.c and ssh.c